drop-cap/library/src/main/java/com/novoda/dropcap/DropCapView.java

```diff
@@ -95,6 +95,8 @@ private void applyCustomAttributes(Context context, AttributeSet attrs) {
             copyTextPaint.setTextSize(copyTextSize);
             copyTextPaint.setColor(copyTextColor);
 
+            String text = typedArray.getString(R.styleable.DropCapView_text);
```

Comment:
That's a good question. Most likely I did it quickly without thinking, I'll update to use the android version, thanks 😃